src/tcpconnect/main.rs

```diff
@@ -0,0 +1,270 @@
+use bcc::core::BPF;
+use std::net::{Ipv4Addr, Ipv6Addr};
+use clap::{App, Arg};
+use failure::Error;
+use std::process;
+use core::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::{ptr, str};
+extern crate ctrlc;
+
+const DEBUG: bool = false;
```

Comment:
I think we should get rid of this. I'm not sure that we need to provide the functionality of printing the bpf code, but I definitely don't like this as a const.

Reply:
I got rid of DEBUG